admin_app/src/app/page.tsx

```diff
@@ -1,9 +1,9 @@
 "use client";
 
```

Comment:
Aesthetics: Similar to login page, during light mode, the background becoms white for me but then the edit pop-up is black/blue. Good colour combos in both light and dark would look nice I think.

Reply:
Added to [AAQ-119](https://idinsight.atlassian.net/browse/AAQ-119) to improve styles